src/python/pants/backend/python/tasks2/select_interpreter.py

```diff
@@ -41,6 +41,11 @@ class SelectInterpreter(Task):
   """Select an Python interpreter that matches the constraints of all targets in the working set."""
 
   @classmethod
+  def register_options(cls, register):
+    super(SelectInterpreter, cls).register_options(register)
+    register('--fixed-interpreter', type=str, help='Use this Python interpreter instead of searching based on constraints', default=None)
```

Comment:
Maybe this option should be in python_setup.py along so it's next to `--interpreter-constraints` ?

Also, if I understand correctly the value here should be a path to a `python` binary? Let's update the help text to clarify this (i.e., what "this Python interpreter" means...)

Reply:
Sure! I thought of doing that but then I don't know how to get the constraints option from there to here. I haven't figured out yet how the current top-level constraints propagate yet.

Reply:
Ah, I need to use the `global_instance` I think. Okay, I'll push an update doing all that!